### PR TITLE
Add opam file

### LINKED ISSUE #9: Add depext section to the opam file
I would open a pull request but I can't find the opam file.

[depext](https://github.com/ocaml/opam-depext) lets you list the system dependencies for an OCaml library, so it would be nice if the opam file had a section like:

```
depexts: [
  [["alpine"] ["freetds-dev"]]
  [["centos"] ["freetds-devel"]]
  [["debian"] ["freetds-dev"]]
  [["fedora"] ["freetds-devel"]]
  [["homebrew"] ["freetds"]]
  [["ubuntu"] ["freetds-dev"]]
]
```

With that, `opam depext freetds` will install the development headers for FreeTDS automatically.

### PR DESCRIPTION
**Pull request by brendanlong**
I got this from:
https://github.com/ocaml/opam-repository/blob/master/packages/freetds/freetds.0.5.1/opam

Then I added:

 - homepage
 - dev-repo
 - bug-reports
 - "./autogen.sh" to build
 - Fedora and Centos sections to depext

Fixes #9

### CODE REVIEW

File: freetds.opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "1.2"
+version: "0.5.1"
```
Comment (Chris00) [inline freetds.opam]: There should be no version in the repository.

File: freetds.opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "ocamlfind" {build}
+]
+depexts: [
+  [["centos"] ["freetds-devel"]]
+  [["debian"] ["freetds-dev"]]
+  [["fedora"] ["freetds-devel"]]
+  [["ubuntu"] ["freetds-dev"]]
+  [["osx" "homebrew"] ["freetds"]]
+]
+install: [make "install"]
```
Comment (Chris00) [inline freetds.opam]: Move the install up — after build — and add a final newline.

File: freetds.opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+tags: [
+  "clib:ct"
+  "clib:sybdb"
+]
+build: [
+  [ "./autogen.sh" ]
+  [
+    "./configure"
+    "--prefix"
+    prefix
```
Comment (Chris00) [inline freetds.opam]: `"--prefix" prefix` on a single line reads better.

File: freetds.opam [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+bug-reports: "https://github.com/kennknowles/ocaml-freetds/issues"
+license: "LGPL-2.1"
+
+tags: [
+  "clib:ct"
+  "clib:sybdb"
+]
+build: [
+  [ "./autogen.sh" ]
+  [
```
Comment (Chris00) [inline freetds.opam]: No need for a new line.

**Comment by brendanlong** [comment]
Fixed.

File: freetds.opam [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+depends: [
+  "ocamlfind" {build}
+]
+depexts: [
+  [["centos"] ["freetds-devel"]]
+  [["debian"] ["freetds-dev"]]
+  [["fedora"] ["freetds-devel"]]
+  [["ubuntu"] ["freetds-dev"]]
+  [["osx" "homebrew"] ["freetds"]]
+]
```
Comment (brendanlong) [inline freetds.opam]: There is a `\n` at the end of this line, but GitHub isn't showing it.

File: freetds.opam [hunk 6]
```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+
+maintainer: "Christophe.Troestler@umons.ac.be"
```
Comment (Chris00) [inline freetds.opam]: Should add a field Authors.  (Sorry, forgot to "push" that one.)

File: freetds.opam (hunk 5 above)
Comment (Chris00) [reply freetds.opam]: It does now (it was complaining there was none before).

**Comment by brendanlong** [comment]
I'm a little confused. This is marked as approved but not merged?

**Comment by Chris00** [comment]
I wanted to add an author field but I did it myself! ;-)

**Pull request merged**